database/schema.js

```diff
@@ -1,7 +1,17 @@
 const mongoose = require('mongoose');
 const Promise = require('bluebird');
+const mongoUrl = 'mongodb://mongo:27017/description'
 
-mongoose.connect('mongodb://localhost/description', {useUnifiedTopology: true, useNewUrlParser: true}).catch(err => console.log(err))
+var connectWithRetry = function() {
+  return mongoose.connect(mongoUrl,{useUnifiedTopology: true, useNewUrlParser: true}, function(err){
+    if (err) {
+      console.log('Failed mongo connect startup retry in 5 sec', err);
+      setTimeout(connectWithRetry,5000);
+    }
+  })
+}
+connectWithRetry();
```

Comment:
I could probably use something like this in my code to seed my db more reliably. 
One thought though, what happens if it still can't connect to the db? It looks like it will loop forever.

Reply:
Oh good catch, I will make an adjustment to that, because yes it will continue to loop right now